### ISSUE TITLE
Cannot use on Mojave

### ISSUE THREAD

**Issue by HotdogWang** [marekkaczkowski/Touch-Bar-iStats#2]
Hi, I already install the istats and import CPU JSON into BTT.
But it doesn't display CPU info on Touch bar.

Then I check the script and found error.
How to fix it ?

Thanks 
![image](https://user-images.githubusercontent.com/12172172/48298675-221cbb80-e4fc-11e8-9785-f7eb97958e65.png)

**Comment by tdauria1** [comment]
I had the same issue. if you change that first line to:

`set temp to do shell script "/bin/bash -lc \"istats scan TCXC PECI CPU --value-only | awk '{print int($1)}'\""`

it should work. I also changed the temp sensor used to the CPU PECI, which is the sensor I use for my fan control. If you want the default, use this instead:

`set temp to do shell script "/bin/bash -lc \"istats cpu --value-only | awk '{print int($1)}'\""`

**Comment by friederbluemle** [comment]
For me the problem was that the `istats` executable was not in `/usr/local/bin`, but in `$HOME/.rbenv/shims`. The following command creates the symlink, which should make the widget work without any modifications:

```
ln -s $(which istats) /usr/local/bin/istats
```

**Comment by ddavidtran** [comment]
I have this problem in the new MacOS Catalina. My istats executable are in `/usr/local/bin`.

I can manually execute the shell script in the terminal except for `"sed s/[a-zA-Z:{},]//g <<< "`

**Comment by emroczka** [comment]
I had a problem with iStats after upgrading to Catalina as well.  My istats executable was in /usr/local/bin as well but I found the problem was that Catalina comes with Ruby v2.6.0 and iStats was expecting to find Ruby v2.3.0.  So here is how I solved the problem:

1. `sudo xcode-select --install`
2. Install rbenv with `brew install rbenv`
3. Add `eval "$(rbenv init -)"` to the end of `~/.zshrc`
4. Install ruby version 2.6.0 using command `rbenv install 2.6.0`
5. Select a ruby version by rbenv `rbenv global 2.6.0`
6. Install iStats again so that it uses the new Ruby `sudo gem install iStats`
7. Test iStats to verify that it is now working `iStats all`
